src/Raven.Studio/typescript/viewmodels/common/notificationCenter/detailViewer/operations/smugglerDatabaseDetails.ts

```diff
@@ -331,6 +359,11 @@ class smugglerDatabaseDetails extends abstractOperationDetails {
         } as smugglerListItem;
     }
 
+    private calcSpeedText(name: string, count: number) {
+        const processingSpeed = this.calculateProcessingSpeed(count, name);
+        return processingSpeed ? `${processingSpeed.toLocaleString()} ${name}/Sec` : smugglerDatabaseDetails.ProcessingText;
```

Comment:
It should be `documents/sec` (lowercase)

Reply:
As suggested by @ml054 now using:` items/sec`